docs/development/getting_started.rst

```diff
@@ -0,0 +1,183 @@
+
+   .. code-block:: bash
+
+       $ source env/bin/activate
+       $ python src/manage.py migrate
+
+6. Create a superuser to access the management interface:
+
+   .. code-block:: bash
+
+       $ python src/manage.py createsuperuser
+
+7. You can now run your installation and point your browser to the address
+   given by this command:
+
+   .. code-block:: bash
+
+       $ python src/manage.py runserver
+
+
+**Note:** If you are making local, machine specific, changes, add them to
+``src/openzaak/conf/includes/local.py``. You can also set certain common
+variables in a local ``.env`` file. You can base these files on the
+example files included in the same directory.
+
+Update installation
+-------------------
+
+When updating an existing installation:
+
+1. Activate the virtual environment:
+
+   .. code-block:: bash
+
+       $ cd open-producten
+       $ source env/bin/activate
+
+2. Update the code and libraries:
+
+   .. code-block:: bash
+
+       $ git pull
+       $ pip install --requirement requirements/dev.txt
+
+3. Update the statics and database:
+
+   .. code-block:: bash
+
+       $ python src/manage.py migrate
+
+
+Testsuite
+---------
+
+To run the test suite:
+
+.. code-block:: bash
+
+    $ python src/manage.py test open_producten
+
+Configuration via environment variables
+---------------------------------------
+
+A number of common settings/configurations can be modified by setting
+environment variables, add them to your ``.env`` file or persist them in
+``src/open_producten/conf/includes/local.py``.
+
+* ``SECRET_KEY``: the secret key to use. A default is set in ``dev.py``
+
+* ``DB_NAME``: name of the database for the project. Defaults to ``open_producten``.
+* ``DB_USER``: username to connect to the database with. Defaults to ``open_producten``.
+* ``DB_PASSWORD``: password to use to connect to the database. Defaults to ``open_producten``.
+* ``DB_HOST``: database host. Defaults to ``localhost``
+* ``DB_PORT``: database port. Defaults to ``5432``.
+
+* ``SENTRY_DSN``: the DSN of the project in Sentry. If set, enabled Sentry SDK as
+  logger and will send errors/logging to Sentry. If unset, Sentry SDK will be
+  disabled.
+
+
+Settings
+========
+
+All settings for the project can be found in
+``src/open_producten/conf``.
+The file ``includes/local.py`` overwrites settings from the base configuration,
+and is only loaded for the dev settings.
+
+.. TODO: remove Celery part below? No references for usage found in project
+
+Running background tasks
+=====================================
+
+We use `Celery`_ as background task queue.
+
+You can run celery worker(s) in a shell to activate the asynchronous task
+queue processing.
+
+To start the background workers executing tasks:
+
+.. code-block:: bash
+
+   $ ./bin/celery_worker.sh
+
+
+To start flower for task monitoring:
+
+.. code-block:: bash
+
+   $ ./bin/celery_flower.sh
+
+
+.. _Celery: https://docs.celeryq.dev/en/stable/
```

Comment:
Is this relevant for this project? We might want to remove this part if Celery is not being used (I could not find any references).